recipes/ragatouille/recipe.yaml

```diff
@@ -0,0 +1,51 @@
+context:
+  version: 0.0.9
+
+package:
+  name: ragatouille
+  version: ${{ version }}
+
+source:
+  - url: https://pypi.org/packages/source/r/ragatouille/ragatouille-${{ version }}.tar.gz
+    sha256: 67957d84686c8b37229f67bb882a9beb5360ba2ddc05aa9590a40eafa201a760
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - pip
+    - setuptools
+  run:
+    - python >=${{ python_min }}
+    - llama-index
+    - faiss-cpu
+    - langchain-core
+    - colbert-ai >=0.2.19
+    - langchain
+    - onnx
+    - srsly
+    - voyager
+    - pytorch >=1.13
+    - fast-pytorch-kmeans
+    - sentence-transformers
+
+tests:
+  - python:
+      python_version: ${{ python_min }}
+      imports:
+        - ragatouille
+      pip_check: false
```

Comment:
Why? This will force all downstream packages to disable it too. If there is a problem that we can fix, let's fix it. If not, we should document the problem and open an  issue.

Reply:
True. The original reason was:

```
 │ │ colbert-ai 0.2.21 requires git-python, which is not installed.
 │ │ colbert-ai 0.2.21 requires ninja, which is not installed.
```

which errors even though I provided them as run dependencies in the `colbert-ai-feedstock`:

https://github.com/conda-forge/colbert-ai-feedstock/blob/a76ce8c4437fdf05cf1e81f36777ead2745f2169/recipe/recipe.yaml#L27-L29

For `git-python`, I'm not sure whether `gitpython` is a drop-in replacement:

https://github.com/stanford-futuredata/ColBERT/blob/7067ef598b5011edaa1f4a731a2c269dbac864e4/setup.py#L25

But the `git-python` on pypi looks sketchy:
https://pypi.org/project/git-python/

Reply:
Can you open an issue in https://github.com/conda-forge/colbert-ai-feedstock/blob/main/recipe/recipe.yaml#L41 about that? Also, please add a comment here so we don't forget.

Reply:
I just created that feedstock myself last week. Happy to fix that but do you have an idea how? Ninja is a run dependency and I'm not quite sure what to do about `git-python`

Reply:
Fix what you can and open issues/notes for the rest. The git-python vs gitpython probably need some metadata fix upstream, not only the recipe.

Reply:
Alright, I fixed the issues related to `colbert-ai`:
https://github.com/conda-forge/colbert-ai-feedstock/pull/1

- gitpython is incorrectly required by upstream (PR to fix upstream [here](https://github.com/stanford-futuredata/ColBERT/pull/388), I patched it in the feedstock)
- `ninja` is required because `colbert-ai` uses `torch`'s ability to load and JIT cpp modules (see [here](https://pytorch.org/docs/stable/cpp_extension.html)) but since the `ninja` package only contains a ninja binary in `bin/ninja` it fails the pip check

Last problem is `faiss-cpu` as a direct dependency. Could this be a similar problem where the package is not recognized as a python package?

Reply:
> Last problem is `faiss-cpu` as a direct dependency. Could this be a similar problem where the package is not recognized as a python package?

No idea. Sorry, my packages are usually quite "vanilla pure python." If that is the case we should add a comment about it and skip pip-check for now.